File: Domovoy/ClientApp/src/api/models/ServiceEntriesCollection.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ApartmentServiceServiceEntryDTO } from './ApartmentServiceServiceEntryDTO';
import type { InformerServiceServiceEntryDTO } from './InformerServiceServiceEntryDTO';
import type { UserServiceServiceEntryDTO } from './UserServiceServiceEntryDTO';

export type ServiceEntriesCollection = {
    userServicesEntries?: Array<UserServiceServiceEntryDTO> | null;
    apartmentServicesEntries?: Array<ApartmentServiceServiceEntryDTO> | null;
    informerServicesEntries?: Array<InformerServiceServiceEntryDTO> | null;
};

File: Domovoy/Models/Services/ServiceEntriesCollection.cs
namespace Domovoy.Models;

public class ServiceEntriesCollection
{
    public List<ServiceEntryDTO<UserService>> UserServicesEntries { get; set; }
    public List<ServiceEntryDTO<ApartmentService>> ApartmentServicesEntries { get; set; }
    public List<ServiceEntryDTO<InformerService>> InformerServicesEntries { get; set; } 
}